index.html

```diff
@@ -918,15 +918,21 @@
     </section>
     <section> <h3>Payment scenarios</h3>
       <p>
-        The user buys goods in a store, and payments options include NFC.
+        Payment scenarios with Web NFC generally do not refer to supporting
+        the payment process itself, but recording the fact of payment with a web
```

Comment:
I am trying to wrap my head around what you wanted to say :)

"but the fact that a payment on the web may happen by the use of NFC technology, which adds the ability to perform secondary NFC actions at the same time"